Source/Constraint.swift

```diff
@@ -417,27 +407,15 @@ public class Constraint {
     
     internal func uninstallFromView() {
         if let view = self.installedOnView {
-            // remove all installed layout constraints
-            var layoutConstraintsToRemove = Array<LayoutConstraint>()
-            if let allObjects = self.installedLayoutConstraints?.allObjects {
-                if let installedLayoutConstraints = allObjects as? Array<LayoutConstraint> {
-                    layoutConstraintsToRemove += installedLayoutConstraints
-                }
-            }
-            
-            if layoutConstraintsToRemove.count > 0 {
+            if let layoutConstraintsToRemove = self.installedLayoutConstraints?.allObjects {
```

Comment:
No need for the previous O(N) operation (that actually runs for each constant removal making it (N^2). I know the constant won't be a lot but removing this actually simplifies the code.